clip2onnx/utils.py

```diff
@@ -0,0 +1,45 @@
+from pathlib import Path
+from typing import Callable
+
+
+def print_available_open_clip_models():
+    import open_clip
+
+    def lookup(mol: Callable[[], list]):
+        for model_name in mol():
+            if tags := open_clip.list_pretrained_tags_by_model(model_name):
+                for tag in tags:
+                    card = {"model": model_name, "tag": tag}
+                    print(f"ModelCard | {card}")
+            card = {"model": model_name, "tag": ""}
+            print(f"ModelCard | {card}")
+
+    lookup(open_clip.list_openai_models)
+    lookup(open_clip.list_models)
+
+
+def request_resource(url: str, save_path: Path):
+    # TODO: Might need to implement a simple download progress bar?
+    import httpx
```

Comment:
This looks like it would be easier to do with the `requests` library, e.g. [like this](https://stackoverflow.com/a/16696317)